android/settings.gradle

```diff
@@ -1,4 +1,8 @@
 rootProject.name = 'LochaMeshChat'
+include ':react-native-vector-icons'
+project(':react-native-vector-icons').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-vector-icons/android')
+include ':react-native-vector-icons'
+project(':react-native-vector-icons').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-vector-icons/android')
```

Comment:
```suggestion
```

Duplicated

Reply:
this is necessary because if I remove this lib the icons can not show.

Reply:
I can see it xD. it was added two times.